src/api/spec/controllers/statistics/maintenance_incidents_controller_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'webmock/rspec'
+
+RSpec.describe Statistics::MaintenanceIncidentsController, type: :controller do
+  describe 'GET #show' do
+    include_context 'a project with maintenance statistics'
+
+    before do
+      login(user)
+
+      get :show, params: { format: :xml, project: project.name }
+    end
+
+    it { is_expected.to respond_with(:success) }
+
+    it 'assigns the project to an instance variable' do
+      expect(assigns[:project]).to be_a(Project)
+    end
+
+    it 'assigns the maintenance_statistics array to an instance variable' do
+      assigns[:maintenance_statistics].each do |maintenance_statistic|
+        expect(maintenance_statistic).to be_a(MaintenanceStatistic)
```

Comment:
What about testing also that the type is something we expect? Because at the moment my code would produce e.g.
```
release_request_HistoryElement::RequestAccepted
```
https://github.com/openSUSE/open-build-service/pull/2507/commits/c0e9b03a236b82cccae72a94fcc85760968daa16#diff-704e02dbad027f297b7e4cc422936eebR24